lesson07/assignment/restaurant/src/components/Carousel.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+ <!-- Bootstrap-Vue Carousel component with 3 images-->
+  <div>
+    <b-carousel
+      id="carousel-1"
+      v-model="slide"
+      :interval="4000"
+      controls
+      indicators
+      background="#ababab"
+      img-width="1024"
+      img-height="480"
+      style="text-shadow: 1px 1px 2px #333;"
+      @sliding-start="onSlideStart"
+      @sliding-end="onSlideEnd"
```

Comment:
Make sure to look at the web console any time you are running your site. I am seeing a lot of errors on your page. It is looking for "onSlideStart" and "onSlideEnd" methods but not finding them. See documentation: https://bootstrap-vue.org/docs/components/carousel